src/bp/core/config/botpress.config.ts

```diff
@@ -103,6 +103,18 @@ export type BotpressConfig = {
      * @example https://botpress.io
      */
     externalUrl?: string
+    session: {
+      /**
+       * @default false
+       */
+      enabled: boolean
+      /**
+       * Number of milliseconds from Date.now() for expiry
+       * Defaults to one hour
+       * @default 3600000
+       */
+      maxAge: number
```

Comment:
I suggest replacing this with a string and use the library ms (like most of our intervals)